plugins/rts54hub/rts54hub.quirk

```diff
@@ -21,3 +21,22 @@ FirmwareSize = 0x30000
 Rts54TargetAddr = 0x20
 Rts54I2cSpeed = 0x2
 Rts54RegisterAddrLen = 0x04
+Rts21xxFwUpdateMode = 1
+
+# Lenovo 2
+[USB\VID_0BDA&PID_5420]
+Plugin = rts54hub
+GType = FuRts54HubDevice
+Vendor = Lenovo
+FirmwareSizeMin = 0x20000
+FirmwareSizeMax = 0x70000
+Children = FuRts54hubRtd21xxDevice|USB\VID_0BDA&PID_5420&I2C_01
+[USB\VID_0BDA&PID_5420&I2C_01]
+Plugin = rts54hub
+Name = HDMI Converter1
+Flags = updatable
+FirmwareSize = 0x70000
+Rts54TargetAddr = 0x20
+Rts54I2cSpeed = 0x2
+Rts54RegisterAddrLen = 0x04
+Rts21xxFwUpdateMode = 2
```

Comment:
is there a minimum firmware version necessary for this feature to work?  Does it need to be probed to check?

Reply:
This feature will controlled by project ID in rts54hub.quirk
if the new project need background update will add the project ID and Rts21xxFwUpdateMode together to rts54hub.quirk